tests/mshevchuk/homework_2.py

```diff
@@ -0,0 +1,95 @@
+import pytest
+
+
+# Дз:
+#  Написати тести які б перевіряли різні помилки в пайтоні наприклад:
+# 5 / 0
+# assert False
+# list()[1000]
+# dict()["abc"]
+# int.foo
+#
+#
+# 2. Спробувати покрити тестами ключові слова, наприклад:
+# not, and, or, in , is,  not in, is not
+# if elif else
+# for in
+# while
+# def
+# компрехеншни,
+# і тд.
+
+def test_zero_division_exception():
+    with pytest.raises(ZeroDivisionError):
+        res = 0 / 0
```

Comment:
no need in assignment here, error still will ocure

`with pytest.raises(ZeroDivisionError):
        0 / 0`